website/sidebarsContributing.ts

```diff
@@ -6,7 +6,7 @@ export default {
       type: 'category',
       label: 'Contributing to React Native',
       collapsed: false,
-      collapsible: true,
+      collapsible: false,
```

Comment:
why are you changing this?

Reply:
There is [only one section in that sidebar](https://reactnative.dev/contributing/overview) which make the collapsible option a bit useless, also wanted to align it with other cases like this, so Community tab, which is currently non-collapsible:
* https://reactnative.dev/community/overview

Reply:
Makes sense 👍